webtranslate/users/github.py

```diff
@@ -29,14 +28,31 @@ def request_teams(name):
         return set()
 
     try:
-        query = """query($org:String!, $user:String!) { organization(login: $org) { teams(first: 100, userLogins: [$user]) { nodes { name } } } }"""
-        variables = { "org": github_organization, "user": name }
-        r = requests.post("https://api.github.com/graphql", headers = { "Authorization": "bearer " + github_org_api_token }, json={'query': query, 'variables': variables})
+        query = """
+        query($org:String!, $user:String!) {
+            organization(login: $org) {
+                teams(first: 100, userLogins: [$user]) {
+                    nodes {
+                        name
+                    }
+                }
+            }
+        }
+        """.replace(
+            "\n", ""
```

Comment:
i wonder if this replacement is actually necessary... surely something like graphql can handle newlines?